lib/SPIRV/SPIRVReader.cpp

```diff
@@ -2798,6 +2798,10 @@ void SPIRVToLLVM::transFunctionAttrs(SPIRVFunction *BF, Function *F) {
     mapValue(BA, &(*I));
     setName(&(*I), BA);
     BA->foreachAttr([&](SPIRVFuncParamAttrKind Kind) {
+      // Skip this function parameter attribute as it will translated among
```

Comment:
Just to clarify, why wasn't it skipped before?

Reply:
Because it wasn't a function parameter attribute (since it was implemented incorrectly)